apps/cookbook/src/app/examples/examples.routes.ts

```diff
@@ -308,6 +308,10 @@ export const routes: Routes = [
     path: 'list-virtual-scroll',
     component: ListVirtualScrollExampleComponent,
     children: [
+      {
+        path: '',
+        redirectTo: 'with-items',
+      },
       {
         path: 'with-items',
         component: ListVirtualScrollItemsExampleComponent,
```

Comment:
Why the additional sub route? 🤔

Reply:
To support popout - works like the list example

Reply:
Ah ok, hadn't seen there where additional examples...
👍🏻